### PR TITLE
add badge icon and list of crowdActionMetadata to badgeDTO

### PR DESCRIPTION
**Pull request by pkhanpara**
First stab at #119 
- added CrowdActionMetadata {id, title} to domain
- added list of CrowdActionMetadata to BadgeDTO
- also added accessiblityOutline string link to the BadgeDTO

Not sure how or where to update the list of CrowdActions associated with the badge. Please review it and leave your suggestions here. I will try to address them promptly.

### CODE REVIEW

**Comment by imReset** [comment]
Please review the changes and inform me if any further adjustments are needed.

File: src/infrastructure/mongo/persistence/badge.persistence.ts [hunk 1]
```diff
@@ -11,5 +11,17 @@ class CrowdActionBadgePersistence implements IBadge {
 
     @Prop({ required: true })
     readonly minimumCheckIns: number;
+
+    @Prop()
+    readonly accessibilityOutline?: string;
```
Comment (MichaelSlatteryDev) [inline src/infrastructure/mongo/persistence/badge.persistence.ts:16]: This should be called icon. Not accessibilityOutline. This applies to everywhere where accessibilityOutline is used in this PR.

**Review by MichaelSlatteryDev — CHANGES_REQUESTED** [review]
Besides the `accessibilityOutline` name it looks all good to me 👍

**Comment by Xazin** [comment]
Thanks @imReset

**Review by MichaelSlatteryDev — APPROVED** [review]

**Pull request merged**